plugins/installer/webinstaller/webinstaller.php

```diff
@@ -251,10 +205,6 @@ private function getChanges()
 
 		if ($ishathor)
 		{
-			if (!$this->isJ25()) {
```

Comment:
re add the code with: https://github.com/zero-24/install-from-web-client/commit/aa764e7a707a479c89ca249ba20de243dec0ed76